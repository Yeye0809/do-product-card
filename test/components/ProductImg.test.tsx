import React from "react";
import  renderer  from "react-test-renderer";
import ProductCard, { ProductImg } from "../../src/componentes";
import { product2 } from '../data/products';

describe('ProductImage', () => {
    test('debe de retornar el link de la imagen', () => {
        
        const wrapper = renderer.create(
            <ProductImg img="https//:holamundo"/>
        );

        expect( wrapper.toJSON() ).toMatchSnapshot();
    });

    test('debe de mostrar el componente con la imagen del producto', () => {

        const wrapper = renderer.create(
            <ProductCard product={ product2 }>
                {
                    () => (
                        <ProductImg />
                    )
                }
            </ProductCard>
        );

        expect( wrapper.toJSON( )).toMatchSnapshot();
      
    })
    
    
})