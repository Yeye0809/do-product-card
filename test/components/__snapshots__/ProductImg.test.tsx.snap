// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage debe de mostrar el componente con la imagen del producto 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="product "
    className="productImg undefined"
    src="./coffee-mug2.png"
  />
</div>
`;

exports[`ProductImage debe de retornar el link de la imagen 1`] = `
<img
  alt="product "
  className="productImg undefined"
  src="https//:holamundo"
/>
`;
